.github/workflows/warn-on-editing-requirements.yml

```diff
@@ -0,0 +1,23 @@
+name: Require core maintainer approval when editing requirements
```

Comment:
can we shorten the name?

Reply:
hahaha yeah seeing it in the checks section i was thinking the name is a bit long

Reply:
i think i can also make it auto re-run when a new review is added